vuepress/docs/next/docs/getting-started/ent-api.md

```diff
@@ -1,4 +1,72 @@
+---
+sidebarDepth: 1
+---
 
 
+# Overview
 
+An API claim informs the Entando Platform that a microfrontend (MFE) or microservice (MS) intends to use the API of another microservice. This request can be sent within a single bundle or across bundles. The abstraction of an API claim eliminates the need to define and manage API endpoints, both in local development and within a running instance.
 
+## Internal vs. External API Claim
+
+An MFE can initiate both internal and external API claims. An internal API claim specifies an MS API existing in the same bundle as the MFE, while an external API claim specifies the MS API of another bundle.
+
+The structure of internal and external API claims are as follows:
+
+**Internal API claim**
+``` json
+{
+    "name": "int-api",
+    "type": "internal",
+    "serviceName": "int-ms"
+}
+```
+**External API claim**
+``` json
+{
+    "name": "ext-api",
+    "type": "external",
+    "serviceName": "ext-ms",
+    "bundle": "registry.hub.docker.com/my-org/my-entando-bundle"
```

Comment:
Open question: can this only reference other bundles within the same project/namespace? We might @w-caffiero-entando to comment but "external" sounds like it could be external to this bundle (as it appears) or external to this project/namespace - say to other projects or even outside kubernetes to other enterprise systems or even to external 3rd party services in the cloud. Based on the format of the bundles I suspect it's only the first case but we should  probably add a clarifying note on the scope.

Reply:
In this context `external` means external to the bundle, but not to the Kubernetes namespace. Indeed, when you execute `ent bundle api add-ext` you have to be connected to an Entando instance and the command will fail if it sees that the service you are specifying in the API claim doesn't exist on the cluster. The external service has to be deployed before adding its API claim.

Reply:
When you execute the command without the flags it searches the available services on the instance, so that you can select one of them interactively:
![image](https://user-images.githubusercontent.com/5459334/182121772-744afde8-3c77-4645-9cb3-d2df0231af92.png)
The command also displays that you are running it on a given instance.